_events/2024-03-23-a-sample-event.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: event
+title: "OLS: a training and mentoring program for the open-research-curious"
+description: "Detailed event description"
```

Comment:
Maybe better to have `tease` here instead of a full description

Reply:
Yes, `detailed` is not the proper word. I intended for us to have this as a brief one-liner describing the event.